.github/workflows/wcag_test.yml

```diff
@@ -0,0 +1,15 @@
+name: WCAG tests
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  call_wcag_test:
+    uses: yext/slapshot-reusable-workflows/.github/workflows/wcag_test.yml@v1
+    secrets:
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+    with:
+      build_script:
+        'npm run storybook & while ! curl http://localhost:6006 -I; do sleep 1; done'
```

Comment:
is there a way to make npm run storybook output a little quieter? if you look at the github action logs there's 1000+ lines for this step https://github.com/yext/search-ui-react/runs/7676458663?check_suite_focus=true#logs

Reply:
updated! now it's <200 lines